tests/warp/warp_test.go

```diff
@@ -423,19 +423,25 @@ func (w *warpTest) deliverAddressedCallToReceivingSubnet() {
 
 	packedInput, err := warp.PackGetVerifiedWarpMessage(0)
 	require.NoError(err)
-	tx := predicate.NewPredicateTx(
-		w.receivingSubnetChainID,
-		nonce,
-		&warp.Module.Address,
-		5_000_000,
-		big.NewInt(225*params.GWei),
-		big.NewInt(params.GWei),
-		common.Big0,
-		packedInput,
-		types.AccessList{},
-		warp.ContractAddress,
-		w.addressedCallSignedMessage.Bytes(),
-	)
+	// Build access list predicate for warp precompile
+	pred := predicate.New(w.addressedCallSignedMessage.Bytes())
+	accessList := types.AccessList{
+		{
+			Address:     warp.ContractAddress,
+			StorageKeys: pred,
+		},
+	}
```

Comment:
While I find this perfectly readable, I feel like it makes the most sense to minimize the diff by including this inline as well